handbook/marketing/index.md

```diff
@@ -2,6 +2,33 @@
 
 <img align="right" src="https://sourcegraphstatic.com/marketing-logo.gif" style="max-height:100%" alt="Sourcegraph Marketing team logo"/>
 
+## How to contact Marketing
+
+### General Marketing requests and feedback
+
+- [Make a request or share an idea](https://form.asana.com/?k=hNaq42PNshdQ1FjjEUKjLA&d=7195383522959)
+- [Sign up for Marketing office hours](https://docs.google.com/spreadsheets/d/1o8rHntP0j6-JFryhc0_g6MKEI6l3dZE7NTKasyQx46M/edit#gid=0)
+- Join the [Marketing team meeting](https://calendar.google.com/event?action=TEMPLATE&tmeid=dWozMjcxcjlnazMxZ2xjcnNkOHZrdjBwNXZfMjAyMTA3MTJUMTUzMDAwWiByZWJlY2NhQHNvdXJjZWdyYXBoLmNvbQ&tmsrc=rebecca%40sourcegraph.com&scp=ALL) (every second week)
+- Slack channel: #marketing
+
+### Who to go to for what
+
+The table below breaks down the capabilities of each team within Marketing. Once you've identified which team is relevant to your query, check out their handbook page or contact them via their Slack channel or contact form. If you're still not sure which team you should contact, please submit a [general request](#General-Marketing-requests-and-feedback) or reach out in #marketing.
+
+| Team                | What we can help with | Slack channels | Submit an idea                                                                            |
+|---------------------|-----------------------|----------------|-------------------------------------------------------------------------------------------|
+| [Brand](brand/index.md)               |                       |                |                                                                                           |
+| [Social Media](brand/index.md)               |  Shaping social strategy under the Brand Team, Social @ Sourcegraph is responsible for the stewardship of the Sourcegraph brand social channels, and enablement of team member social profiles. We are accountable for the organic editorial calendar and partner with other departments across the organization to provide organic social promotion.                       |  #social-media-action & #social-media-posts              |  Send requests, ideas, and industry updates to the #social-media-action channel                                                                                         |
```

Comment:
@kristen-sundberg Thank you so much for adding your details here ✨ Thinking of this from the perspective of someone outside marketing coming to this page to find out who can assist them with an ask, maybe we could word this more explicitly? The column is "What we can help with" so the answer could be "Promoting new content and events via Sourcegraph branded social channels, guidance on personal social media profiles and engagement, ...." I'll leave it to you, but hopefully you get the idea!

Reply:
Hi Rebecca! I'm adding my info here again. It's not letting me reply to your comment anymore. This is for Brand: 

Slack channels: #brand-creative #swag
Creative requests: https://form.asana.com?k=Zv_DWGDitZCYr0FXY3bvHg&d=7195383522959
Swag requests:https://form.asana.com?k=ysBKSzSA2xjRifqqEvFbJQ&d=7195383522959

Thank you again for doing this <3